src/collections/blog/2022-01-07-Recap-KubeCon-China-TAG-Network-talk/index.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: "Recap KubeCon China 2021: TAG Network Deep-Dive"
+date: 2022-01-07 10:30:05 -0530
+author: Deepak Gupta
+thumbnail: ./tag-network-kubecon.png
+category: "Events"
+tags:
+ - CNCF
+ - Meshery
+ - Service Mesh Performance
+ - Nighthawk
+published: true
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+import Blockquote from "../../../reusecore/Blockquote";
+import Projects from "./projects.png";
+import Gnh from "./gnh-meshery.png";
+import Smp from "./smp-meshery.png";
+import Smi from "./smi-meshery.png";
+import Patterns from "./patterns.png";
+import Catalog from "./catalog.png";
+import slidesTAG from "../../events/kubecon-china-2021/cncf-tag-network-and-service-mesh-working-group-kubecon-china-2021-lee-calcote-ken-owens.pdf";
+import { Link } from "gatsby";
+
+
+
+
+
+<BlogWrapper>
+
+<div class="intro ">
+<p>
+    The CNCF TAG Network Co-Chairs <Link to="/members/lee-calcote">Lee Calcote</Link> and Ken Owens gave a presentation entitled Intro and Deep Dive into CNCF TAG Network and CNCF Service Mesh Working Group at KubeCon China 2021.
+</p>
+</div>
+
+<p>
+    With the rise of microservice-based distributed systems, networking has never been more important in ensuring the smooth operation of cloud native deployments. 
+Cloud native architectures call for horizontally scalable, distributed systems design with microservices proliferated from Cloud to Edge. Technical Advisory Group (TAG) Network’s objective is to enable widespread and effective development, deployment, and operation of resilient and intelligent network systems in cloud native environments, keeping an eye on the needs of workloads, developers who design them, and operators who manage them.
```

Comment:
```suggestion
Cloud native architectures call for horizontally scalable, distributed systems design with microservices proliferated from Cloud to Edge. **Technical Advisory Group (TAG)** Network’s objective is to enable widespread and effective development, deployment, and operation of resilient and intelligent network systems in cloud native environments, keeping an eye on the needs of workloads, developers who design them, and operators who manage them.
```
Will it be alright if abbreviation can be made bold? This may or may not be included. But I thought it is good practice.

Reply:
Nope. There's no published writing style that includes this practice.

Reply:
If this suggestion as incorporated, it should be reverted.

Reply:
Gotit 👍🏽  I read it [here ](https://developers.google.com/tech-writing/one/words#use_acronyms_properly) sometime back 😅

Reply:
Oh, I see. Very good. 

Yes, well-accepted and we should enforce: "On the initial use of an unfamiliar acronym within a document or a section, spell out the full term, and then put the acronym in parentheses."
We'll pass on Google's other boldface suggestion: "Put both the spelled-out version and the acronym in boldface."

Reply:
Aight sire!

Reply:
I think well that you were able to show reference justifying this style change, @Mr-DG-Wick. Good on you for this.